CMakeLists.txt

```diff
@@ -1,43 +1,69 @@
-# Copyright (c) 2019 Linaro
```

Comment:
Not related to this code line, but to the commit message:
> This commit retrieves information about the toolchain from the Zephyr ENV variables, rather than hard-coding specific paths in.

It's not `ENV variables`, cause they will be fetched using `$ENV{<VAR_NAME>}`, it's just Zephyr CMake variables.